### ISSUE TITLE
Allow grouping of recipients

### ISSUE THREAD

**Issue by marceloomens** [globaleaks/GlobaLeaks#1377]
We'd like to make it possible to group recipients in a context. The use case in the the case of Publeaks is that we'd like to be able to display national media and regional media separately from each other.

**Comment by evilaliv3** [comment]
@vecna an other aspect discussed with @marceloomens and the great goldilocks team for publeaks is to have the possibility to group receivers.

they will use such a feature in order to split receivers in multiple groups (just for a matter of visualization); e.g. local medias, national medias; and they would like that on the submission stage the whistleblower will have the possibility to limit the selection per group.

@marceloomens this is somehow the requirement that we highlighted for the possibility to have receivers  divided in levels of priviledge; e.g. first level receivers (that receiver the data first) and second level receivers (that can receive data if forwarded by first level receivers)

in the general to implement both things indipendently all should need a table like the following
- id: an id of the group
- label [subject to internationalization that would allow to say what is this group for when shown on public interfaces]
- presentation_order: the number that gives order of the group among the others
- receivers: a list of receivers in the group
- a set of options 

somehow we should undertstand if this simple table can be used to obtain both the scores; i've to think a little if the two thing may be merged or not (i dont' want to have a simple db but a totally complex logic around).

**Comment by evilaliv3** [comment]
\cc @fpietrosanti

**Comment by vecna** [comment]
for history records, "group" has been one of the first feature we removed to simplify the software. for me this has to be only a "submission interface visualization feature", but then all in the system has to works in the same way. multilevel receiver is something we can approach separately.

**Comment by evilaliv3** [comment]
i agree. but anyhow the feature needed for publeaks would require a db structure as the one described above or something as it should be possible to divide anyhow receivers in groups, and a group should have the possibility to have assigned an order and a label.

i will think a little about this.

**Comment by fpietrosanti** [comment]
This feature is not related to https://github.com/globaleaks/GlobaLeaks/issues/1196 right?

Sent from mobile

**Comment by marceloomens** [comment]
@fpietrosanti I think that's a feature which is interesting for Publeaks also, but it's not what I mean.

**Comment by yevon** [comment]
Hi any news on this one? It would be interesting that kind of user grouping for managing case security by user groups. If not we will have to share users and treat as groups.

**Comment by evilaliv3** [comment]
I'm sorry yevon, i'm noticing this only now. Up to date we have no clear idea on this matter.

do you have more up to date suggestions?
